### PR TITLE
mkfs.exfat.8: document default --boundary-align & --cluster-size

### LINKED ISSUE #220: mkfs: Documenting default for --boundary-align and --cluster-size
mkfs.exfat.8 currently has a recommendation table from the SD association for these two options, but does not actually describe what this program does by default via https://github.com/exfatprogs/exfatprogs/blob/3228312ffeff051870de9178b893f5676ed5e773/lib/libexfat.c#L138-L139

Specifically, the default boundary alignment is fixed at 1 MiB, and the default cluster size is set in https://github.com/exfatprogs/exfatprogs/blob/3228312ffeff051870de9178b893f5676ed5e773/lib/libexfat.c#L120-L121

### PR DESCRIPTION
**Pull request by Artoria2e5**
This closes #220. This tbl stuff is new to me...

### CODE REVIEW

File: manpages/mkfs.exfat.8 [hunk 1]
```diff
@@ -67,12 +67,23 @@ _
...
 .BR \-c ", " \-\-cluster\-size =\fIsize\fR
 Specifies the cluster size of the exFAT file system.
 The \fIsize\fR argument is specified in bytes or may be specified with
 \fBm\fR/\fBM\fR suffix for mebibytes or \fBk\fR/\fBK\fR suffix for kibibytes
 and must be a power of two.
+The default value is described in the following table:
+.TS
+center;
+cb1s6cb,nnn.
+Card Capacity Range	Cluster Size
```
Comment (Artoria2e5) [inline manpages/mkfs.exfat.8:81]: Not quite "card"... maybe this instead?
```suggestion
Filesystem Size Range	Cluster Size
```

File: manpages/mkfs.exfat.8 [hunk 2]
```diff
@@ -67,12 +67,23 @@ _
 >128 GiB	\[<=]512 GiB	256 KiB	32 MiB
 >512 GiB	\[<=]2 TiB	512 KiB	64 MiB
 .TE
+The default is always 1 MiB.
```
Comment (Artoria2e5) [inline manpages/mkfs.exfat.8:70]: Might be a good idea to have some rationale here. Just the obvious "not every block device is an SD card" could do.

**Comment by Artoria2e5** [comment]
Wow, SD Part 2 is some... interesting reading.

**Comment by namjaejeon** [comment]
@hclee How about this ?

**Comment by hclee** [comment]
@namjaejeon
This looks good to me.

**Comment by namjaejeon** [comment]
Applied it to #exfat-next, Thanks for your patch!

**Pull request closed without merging**